xml/System.Web.UI.WebControls/LinqDataSourceView.xml

```diff
@@ -1936,9 +2079,24 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <summary>Saves the changes to view state for the <see cref="T:System.Web.UI.WebControls.LinqDataSourceView" /> control since the time that the page was posted back to the server.</summary>
+        <returns>The object that contains the changes to the <see cref="T:System.Web.UI.WebControls.LinqDataSourceView" /> control's view state, or null (Nothing in Visual Basic) if there is no view state associated with the object.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks  
+The SaveViewState method is primarily used by control developers.
```

Comment:
I think "SaveViewState" needs code brackets to prevent translation.

Below the "ViewState" needs code brackets to prevent translation